proxy/dialers/postgres/hook.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package postgres adds a 'cloudsqlpostgres' driver to use when you want
+// to access a Cloud SQL Database via the go database/sql library.
+// It is a wrapper over the driver found at github.com/lib/pq.
+// To use this driver, you can look at an example in
+// postgres_test package in the hook_test.go file
+package postgres
```

Comment:
Doing that also requires a blank import of this package, which is tedious. It's nice to have a function to get the *sql.DB in this package since it will be a simpler explicit import (which are easier to understand and harder to forget). Plus, then your helper can call Ping and help prevent the giant misconception that sql.Open is proactive rather than lazy-dialing.

Also, that's a fairly annoying line to write. We want this library to be easy for people who have not used this driver before. It's nice to give them a Go function to call which will construct the DSN correctly rather than asking them to read a bunch of docs outside of this repo.

Definitely not asking you to remove what you have already, what you have is good to have. But having convenience functions is nice for people using our library, since I'd like this to be a convenience function.